clients/fides-js/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,69 @@
+import { h } from "preact";
+import { I18n } from "../lib/i18n";
+import { useI18n } from "../lib/i18n/i18n-context";
+import { debugLog, FidesInitOptions } from "../fides";
+import MenuItem from "./MenuItem";
+
+interface LanguageSelectorProps {
+  i18n: I18n;
+  options: FidesInitOptions;
+}
+
+const LanguageSelector = ({ i18n, options }: LanguageSelectorProps) => {
+  const { currentLocale, setCurrentLocale } = useI18n();
+
+  const handleLocaleSelect = (locale: string) => {
+    if (locale !== i18n.locale) {
+      i18n.activate(locale);
+      setCurrentLocale(locale);
+      document.getElementById("fides-button-group")?.focus();
```

Comment:
can we store this str as a const either at top of this file or in a shared const / types file?

Reply:
good call